pkgs/applications/networking/flexget/default.nix

```diff
@@ -35,6 +35,8 @@ python3Packages.buildPythonApplication rec {
     # See https://github.com/Flexget/Flexget/blob/master/requirements.in
     APScheduler
     beautifulsoup4
+    click
+    colorama
```

Comment:
`colorama` is already a dependency of logru
```suggestion
```
Besides that, LGTM

Reply:
It's only a checkInputs dependency of loguru, not propagated. And we really shouldn't be rely on propagated deps for something that's directly imported by flexget, it's fragile as shown by precisely this PR.

Without the explicit colorama dep:
```
$ /nix/store/pai4jg9cpb2ghjb0s432nl4qbmbgwqji-flexget-3.3.8/bin/flexget daemon start
Traceback (most recent call last):
  File "/nix/store/pai4jg9cpb2ghjb0s432nl4qbmbgwqji-flexget-3.3.8/bin/.flexget-wrapped", line 6, in <module>
    from flexget import main
  File "/nix/store/pai4jg9cpb2ghjb0s432nl4qbmbgwqji-flexget-3.3.8/lib/python3.9/site-packages/flexget/__init__.py", line 10, in <module>
    from flexget import log  # noqa
  File "/nix/store/pai4jg9cpb2ghjb0s432nl4qbmbgwqji-flexget-3.3.8/lib/python3.9/site-packages/flexget/log.py", line 15, in <module>
    import colorama
ModuleNotFoundError: No module named 'colorama'
```